parachain/primitives/core/src/types.rs

```diff
@@ -24,6 +24,9 @@ pub struct Message {
 	pub verification: VerificationInput,
 }
 
+#[derive(Clone, Encode, Decode, Default, PartialEq, RuntimeDebug)]
+pub struct Messages(pub AppId, pub Vec<Message>);
```

Comment:
Would prefer the name `MessageBatch` (even over other suggestions like `BulkMessages`, etc), as `Messages` could be confused for plain list of messages.